src/test/java/com/google/api/codegen/testdata/java_test_test_library.baseline

```diff
@@ -199,12 +267,19 @@ public class LibraryServiceTest {
   @Test
   @SuppressWarnings("all")
   public void listBooksTest() {
+    ListBooksResponse response = ListBooksResponse.newBuilder()
+      .build();
+    List<GeneratedMessage> responses = new ArrayList<>();
```

Comment:
There is an important way that the sample code init fields and the tests need to differ: sample code init fields should initialize as little as possible; tests should initialize as much as possible (to make sure it gets plumbed through). This test doesn't really verify much, because an empty list can't be distinguished between different API method calls.